distutils/command/install.py

```diff
@@ -190,6 +190,8 @@ class install(Command):
 
     negative_opt = {'no-compile' : 'compile'}
 
+    # Allow Fedora to add components to the prefix
+    _prefix_addition = getattr(sysconfig, '_prefix_addition', "")
```

Comment:
Since this is no longer patched globally, I'm tempted to move it inline with the other code rather than exposing a class attribute. WDYT?

Reply:
Done in 902e204.